paper/paper.md

```diff
@@ -0,0 +1,197 @@
+In order to allow the integration of learners that are not available on `CRAN`, the package is hosted on the [`mlr` R-universe](https://mlr-org.r-universe.dev/).
+By providing a standardized interface and comprehensive metadata for each learner, mlr3extralearners enhances the FAIRness (findability, accessibility, interoperability, and reusability) of machine learning algorithms within the R ecosystem [@wilkinson2016fair]).
+
+# Statement of Need
+
+Machine learning often requires practitioners to navigate a diverse array of modeling problems, each with unique demands such as performance, interpretability, or compatibility with specific data types and tasks.
+To address this challenge, packages like `caret` [@caret] and `parsnip` [@parsnip] from the `tidymodels` ecosystem have historically provided unified interfaces for simplifying model experimentation [@tidymodels].
+For instance, `parsnip` provides a clean and consistent way to define models, enabling users to experiment with different algorithms without dealing with the nuances of underlying package syntax.
+Similarly, the `mlr3` ecosystem aims to streamline model selection and experimentation, making it a versatile toolbox for machine learning in R.
+
+Within this ecosystem, `mlr3extralearners` plays a crucial role by providing a comprehensive collection of external machine learning algorithms integrated into the `mlr3` framework.
+This ensures that users can access a wide variety of learners to meet their specific needs,
+and choose the most appropriate learner for their specific problem.
+While connecting new learners to `mlr3` is straightforward and can be done on a per-need basis, integrating them into `mlr3extralearners` benefits the broader community by avoiding redundant effort and ensuring accessibility for all users.
+Additionally, contributions to `mlr3extralearners` are reviewed by the package maintainers, providing a layer of quality assurance.
+This review process ensures that integrated learners work as expected and adhere to the high standards of the `mlr3` ecosystem.
+
+Beyond its utility for users, `mlr3extralearners` also offers significant advantages for developers of machine learning packages.
+By integrating a new algorithm into the `mlr3` ecosystem, developers can immediately make their methods accessible to a wider audience.
+This integration facilitates seamless tuning [@mlr3tuning] and preprocessing [@mlr3pipelines2021] through the broader `mlr3` framework, enhancing the usability and impact of their work.
+
+# Features
+
+The core functionality of `mlr3extralearners` is to integrate new learners into the `mlr3` ecosystem, allowing users to access a wide array of learning algorithms through a unified syntax and standardized interface.
+However, the advantages of `mlr3extralearners` go well beyond simple integration.
+
```

Comment:
Why don't we add a few codes examples like [list_mlr3learners()](https://mlr3extralearners.mlr-org.com/reference/list_mlr3learners.html) to show all availble learners? Or show the unified interface with `lrn("classif.gbm")`?

Reply:
We have the link to the website with all learners, this covers the first. The second we could, let's see what Bernd also thinks when he reads it.